### PR TITLE
Added Privacy Sandbox references.md

### PR DESCRIPTION
**Pull request by EmLauber**
plus linked to front channel logout spec.

### CODE REVIEW

**Comment by EmLauber** [comment]
@hectormmg @peterzenz can you monitor for feedback and update while I'm out? I think this should be enough to address recent customer feedback.

**Comment by hectormmg** [comment]
@EmLauber will do, thanks!

**Comment by Jak-MS** [comment]
@OwenRichards1 
- Can you review this PR?
- IMPORTANT: When this content is ready to merge, you must add `#sign-off` in a comment or the approval may get overlooked.


#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: docs/identity-platform/reference-third-party-cookies-spas.md [hunk 1]
```diff
@@ -17,14 +17,12 @@ ms.topic: conceptual
...
+## What is Intelligent Tracking Protection (ITP) and Privacy Sandbox?
 
-Apple Safari has an on-by-default privacy protection feature called [Intelligent Tracking Protection](https://webkit.org/tracking-prevention-policy/), or _ITP_. ITP blocks "third-party" cookies, cookies on requests that cross domains.
+Apple Safari has an on-by-default privacy protection feature called [Intelligent Tracking Protection](https://webkit.org/tracking-prevention-policy/), or _ITP_. Chrome has a browser privacy intiative named the [Privacy Sandbox](https://developers.google.com/privacy-sandbox/overview). These initiatives encompass many different browser privacy efforts by the browsers and have different timelines. Both efforts block "third-party" cookies, cookies on requests that cross domains. Safari and Brave block third-party cookies by default, and Chrome announced that they'll start blocking third-party cookies by default [in the future](https://privacysandbox.com/open-web/#the-privacy-sandbox-timeline).The Privacy Sandbox includes changes to [partitioned storage](https://developers.google.com/privacy-sandbox/3pcd/storage-partitioning) as well as third-party cookie blocking.
```
[the remark this answers is not in the document]
Comment (OwenRichards1) [reply docs/identity-platform/reference-third-party-cookies-spas.md:22]: > These initiatives encompass many different browser privacy efforts by the browsers and have different timelines. 

Just wondering can we add some further clarity to this statement, currently i'm not sure what it is trying to say.

**Review by OwenRichards1 — APPROVED** [review]
LGTM! Few small changes.

**Pull request merged**